dim-testsuite/t/container-list.t

```diff
@@ -12,7 +12,25 @@ INFO - Creating container 10.0.0.0/8 in layer3domain one
 $ ndcli create container 11.0.0.0/8 layer3domain one
 INFO - Creating container 11.0.0.0/8 in layer3domain one
 $ ndcli list containers
```

Comment:
We should add more containers, pools and delegations here to get all variants of possible output.

```
$ ndcli create container 10.0.4.0/23 layer3domain default
$ ndcli create pool a layer3domain default
$ ndcli modify pool a add subnet 10.0.5.0/24
$ ndcli modify pool a get delegation 27
```

When doing `list containers layer3domain default 10.0.5.0/25` it should then return all found parents and children in that space.

Reply:
You are right. Thank you :)